src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/VedtaksBegrunnelsePanel.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+
+import styled from 'styled-components';
+
+import navFarger from 'nav-frontend-core';
+import Ekspanderbartpanel from 'nav-frontend-ekspanderbartpanel';
+import Hjelpetekst from 'nav-frontend-hjelpetekst';
+import { Element, Normaltekst } from 'nav-frontend-typografi';
+
+import { useBehandling } from '../../../../context/BehandlingContext';
+import { Behandlingstype } from '../../../../typer/behandling';
+import { IUtbetalingsperiode, IUtbetalingsperiodeDetalj } from '../../../../typer/beregning';
+import { periodeToString, TIDENES_MORGEN } from '../../../../typer/periode';
+import { IRestPersonResultat } from '../../../../typer/vilkår';
+import { formaterBeløp, formaterPersonIdent, isoStringToDayjs } from '../../../../utils/formatter';
+import { sisteDagInneværendeMåned } from '../../../../utils/tid';
+import VedtakBegrunnelserMultiselect from './VedtakBegrunnelserMultiselect';
+
+interface IVedtakBegrunnelserTabell {
+    utbetalingsperiode: IUtbetalingsperiode;
+    personResultater: IRestPersonResultat[];
+    behandlingsType: Behandlingstype;
+}
+
+const StyledEkspanderbartpanel = styled(Ekspanderbartpanel)`
+    margin-bottom: 1.5rem;
+    width: 49rem;
+
+    .ekspanderbartPanel__hode {
+        padding-top: 0;
+        padding-bottom: 0;
+    }
+    .ekspanderbartPanel__innhold {
+        padding: 1rem;
+    }
+`;
+
+const UtbetalingsperiodepanelTittel = styled.p`
+    display: flex;
+    align-items: center;
+    text-align: center;
+
+    .typo-normal {
+        margin-left: 1.5rem;
+    }
+`;
+
+const UtbetalingsperiodepanelBody = styled.div`
+    margin-left: 0.625rem;
+    display: grid;
+    grid-template-columns: 5fr 4fr;
+`;
+
+const UtbetalingsperiodeDetalj = styled.div`
+    display: flex;
+    flex-direction: row;
+
+    .typo-normal {
+        margin-right: 1.5rem;
+    }
+`;
+
+const HjelpetekstWrapper = styled.button`
```

Comment:
Er denne lik den andre som brukes for tittel? Isåfall kan man kanskje trekke det ut til en egen komponent?